test/karma-files.json

```diff
@@ -66,6 +66,7 @@
 	"code/indicators/roc.src.js",
 	"code/indicators/rsi.src.js",
 	"code/indicators/stochastic.src.js",
+	"code/indicators/aroon.src.js",
```

Comment:
Not a big deal, but could you (unless this should be here) place this in alphabetical order?